packages/prop-house-backend/src/proposal/proposal.entity.ts

```diff
@@ -58,6 +59,7 @@ export class Proposal extends SignedEntity {
   @BeforeUpdate()
   updateVoteCount() {
     this.voteCount = this.votes.reduce((acc, vote) => {
+      if (vote.signatureState !== SignatureState.VALIDATED) return;
```

Comment:
Should be `if (vote.signatureState !== SignatureState.VALIDATED) return acc;`, right?

Reply:
yep! thanks for catching.